LoopKitUI/Views/Settings Editors/InsulinModelSelection.swift

```diff
@@ -81,73 +84,89 @@ public struct InsulinModelSelection: View, HorizontalSizeClassOverride {
-                        text: insulinModelSettingDescription,
-                        informationalContent: {
-                            // TODO: Implement informational content
-                            Text("Not implemented")
-                        }
-                    )
-                    .padding(4)
-                    .padding(.top, 4)
-
-                    VStack {
-                        InsulinModelChartView(
-                            chartManager: chartManager,
-                            glucoseUnit: glucoseUnit,
-                            selectedInsulinModelValues: selectedInsulinModelValues,
-                            unselectedInsulinModelValues: unselectedInsulinModelValues,
-                            glucoseDisplayRange: endingGlucoseQuantity...startingGlucoseQuantity
-                        )
-                        .frame(height: 170)
-
-                        CheckmarkListItem(
-                            title: Text(InsulinModelSettings.exponentialPreset(.humalogNovologAdult).title),
-                            description: Text(InsulinModelSettings.exponentialPreset(.humalogNovologAdult).subtitle),
-                            isSelected: isSelected(.exponentialPreset(.humalogNovologAdult))
-                        )
-                        .padding(.vertical, 4)
+            content
+            .navigationBarItems(leading: dismissButton)
+        }
+    }
+    
+    private var content: some View {
+        List {
+            Section {
+                SettingDescription(
+                    text: insulinModelSettingDescription,
+                    informationalContent: {
+                        TherapySetting.insulinModel.helpScreen()
                     }
+                )
+                .padding(4)
+                .padding(.top, 4)
+
+                VStack {
+                    InsulinModelChartView(
+                        chartManager: chartManager,
+                        glucoseUnit: glucoseUnit,
+                        selectedInsulinModelValues: selectedInsulinModelValues,
+                        unselectedInsulinModelValues: unselectedInsulinModelValues,
+                        glucoseDisplayRange: endingGlucoseQuantity...startingGlucoseQuantity
+                    )
+                    .frame(height: 170)
 
                     CheckmarkListItem(
-                        title: Text(InsulinModelSettings.exponentialPreset(.humalogNovologChild).title),
-                        description: Text(InsulinModelSettings.exponentialPreset(.humalogNovologChild).subtitle),
-                        isSelected: isSelected(.exponentialPreset(.humalogNovologChild))
+                        title: Text(InsulinModelSettings.exponentialPreset(.humalogNovologAdult).title),
+                        description: Text(InsulinModelSettings.exponentialPreset(.humalogNovologAdult).subtitle),
+                        isSelected: isSelected(.exponentialPreset(.humalogNovologAdult))
```

Comment:
This diff is also showing up weirdly, nothing has actually changed (I copy-based most of the `body` into `content`)

Reply:
I think since the NavigationView was pulled up, the indentation diff is confusing it!